package.json

```diff
@@ -11,7 +11,7 @@
     "semantic-release": "semantic-release --debug"
   },
   "engines": {
-    "node": ">=0.10.0 <0.14"
+    "node": ">=0.10.0 <0.15"
```

Comment:
Ist die Version eigentlich 0.15? Wenn ich https://nodejs.org/en/about/releases/ dort schaue ist es ja eigentlich 15.x. Ich hab versucht zu googlen. Aber man findet tatsächlich immer wieder beide Schreibweisen. Meinungen?

Reply:
wir machen das so, wie es auch als Beispiel in der npm doku steht, das erscheint mir sinnvoll: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines

Reply:
Ja das hab ich auch gefunden. Also weißt du da aber sonst auch nicht mehr.